src/client/src/system/dimensions.jsx

```diff
@@ -0,0 +1,140 @@
+/*eslint-disable */
```

Comment:
agreed, will amend.

On Monday, 25 April 2016, Keith Woods <notifications@github.com> wrote:

> In src/client/src/system/dimensions.jsx
> <https://github.com/AdaptiveConsulting/ReactiveTraderCloud/pull/302#discussion_r60889533>
> :
>
> > @@ -0,0 +1,140 @@
> > +/*eslint-disable */
>
> I think we need to add the licencing information as per the initial
> projects licencing agreement. Additionally can we write a note as to why
> we've forked it locally.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/AdaptiveConsulting/ReactiveTraderCloud/pull/302/files/06adbf723df5998eaf38c3d69099f6f092d910ba#r60889533>
>


-- 
Dimitar Christoff

"JavaScript is to JAVA what hamster is to ham"
@D_mitar - https://github.com/DimitarChristoff